### PR TITLE
Update search results page layout

### PR DESCRIPTION
**Pull request by Mitso**

### CODE REVIEW

**Comment by Mitso** [comment]
@saeedmb8 pease review 

screenshot 2017-03-07 18 46 29

**Comment by saeedmb8** [comment]
👍

File: iogt/templates/search/search_results.html [hunk 1]
```diff
@@ -6,30 +6,53 @@
...
+      {% if ancestor.sectionpage.image %}
+        <a href="{% pageurl page %}" class="search__result-item article-list__image-content">
+          {% if  page.image %}
+            {% image page.image width-100 height-100 class="article-list__image article-list__image--small" %}
+          {% endif %}
+        </a>
+        <a href="{% pageurl page %}" class="search__result-item article-list__inner-content">
+          <h3 class="search--result__section-title search--result__section-title{{page.get_parent_section.get_effective_extra_style_hints}}">
+            {{ancestor.title}}</h3>
+          <h1 class="search--result__title">{{page.title}}</h3>
```
Comment (smarzban) [inline iogt/templates/search/search_results.html]: The opening and the closing tags are not same here

**Comment by smarzban** [comment]
left a tiny comment otherwise 👍

**Pull request merged**